scripts/build_lilypond.sh

```diff
@@ -15,6 +15,15 @@ LILYPOND_SRC="$LILYPOND/src"
 LILYPOND_BUILD="$LILYPOND/build"
 LILYPOND_INSTALL="$LILYPOND/install"
 
+if [[ $VERBOSE -ne "0" ]]; then
```

Comment:
This is bash syntax AFAIK, the portable syntax should be `[ $VERBOSE != 0 ]`

Reply:
Good catch. Will fix.

Reply:
Fixed in commit f0a658b